docs/Formula-Cookbook.md

```diff
@@ -677,12 +677,16 @@ Generally we'd rather you were specific about what files or directories need to
 | **`sbin`**            | `#{prefix}/sbin`                               | `/usr/local/Cellar/foo/0.1/sbin`                            |
 | **`share`**           | `#{prefix}/share`                              | `/usr/local/Cellar/foo/0.1/share`                           |
 | **`pkgshare`**        | `#{prefix}/share/foo`                          | `/usr/local/Cellar/foo/0.1/share/foo`                       |
+| **`elisp`**           | `#{prefix}/share/emacs/site-lisp/foo`          | `/usr/local/Cellar/foo/0.1/share/emacs/site-lisp/foo`       |
+| **`frameworks`**      | `#{prefix}/Frameworks`                         | `/usr/local/Cellar/foo/0.1/Frameworks`                      |
+| **`kext_prefix`**     | `#{prefix}/Library/Extensions`                 | `/usr/local/Cellar/foo/0.1/Library/Extensions`              |
 | **`zsh_function`**    | `#{prefix}/share/zsh/site-functions`           | `/usr/local/Cellar/foo/0.1/share/zsh/site-functions`        |
 | **`fish_function`**   | `#{prefix}/share/fish/vendor_functions`        | `/usr/local/Cellar/foo/0.1/share/fish/vendor_functions`     |
 | **`bash_completion`** | `#{prefix}/etc/bash_completion.d`              | `/usr/local/Cellar/foo/0.1/etc/bash_completion.d`           |
 | **`zsh_completion`**  | `#{prefix}/share/zsh/site-functions`           | `/usr/local/Cellar/foo/0.1/share/zsh/site-functions`        |
 | **`fish_completion`** | `#{prefix}/share/fish/vendor_completions.d`    | `/usr/local/Cellar/foo/0.1/share/fish/vendor_completions.d` |
 | **`etc`**             | `#{HOMEBREW_PREFIX}/etc`                       | `/usr/local/etc`                                            |
+| **`pkgetc`**          | `#{HOMEBREW_PREFIX}/etc/foo`                   | `/usr/local/etc/foo`                                        |
```

Comment:
```suggestion
| **`pkgetc`**          | `#{HOMEBREW_PREFIX}/etc/#{name}`                   | `/usr/local/etc/foo`                                        |
```

Can you also correct `doc` and `pkgshare` similarly?

Reply:
Good point. Done.